diff --git a/unionfs/src/CMakeLists.txt b/unionfs/src/CMakeLists.txt
index 7f3703089..3d8359677 100644
--- a/unionfs/src/CMakeLists.txt
+++ b/unionfs/src/CMakeLists.txt
@@ -4,7 +4,7 @@ set(UNIONFS_SRCS unionfs.c opts.c debug.c findbranch.c readdir.c
     fuse_ops.c)
 set(UNIONFSCTL_SRCS unionfsctl.c)
 
-SET(_COMMON_FLAGS "-pipe -W -Wall -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64")
+SET(_COMMON_FLAGS "-static -pipe -W -Wall -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64")
 SET(CMAKE_C_FLAGS_RELWITHDEBINFO "-O2 -g ${_COMMON_FLAGS}")
 SET(CMAKE_C_FLAGS_RELEASE "-O2 ${_COMMON_FLAGS}")
 SET(CMAKE_C_FLAGS_DEBUG "-O0 -g -DDEBUG ${_COMMON_FLAGS}")
